content/search-github/github-code-search/using-github-code-search.md

```diff
@@ -40,11 +40,15 @@ For more information about the search syntax of code search, see "[AUTOTITLE](/s
 ## Creating and managing saved searches
 
 1. In the top navigation of {% data variables.product.prodname_dotcom_the_website %}, click the search bar and type `saved:`.
-1. Under the search bar, the "Saved searches" section should now appear. Click {% octicon "plus-circle" aria-hidden="true" %} **Create saved search**.
+![GitHub top navigation, user typing 'saved:' in search bar.](/assets/images/help/search/navigatetosearch.png)
+1. Under the search bar, the "Saved queries" section should now appear. Click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
+!['Saved queries' section, user clicking 'Manage saved searches'.](/assets/images/help/search/selectmanagesavedsearch.png)
 1. In the pop-up window, fill out the name you want for your query and the query that you want to save. Click **Create saved search**.
-1. If you click again on the search bar, you can now see your saved search in the "Saved searches" section under the search bar. Clicking on a saved search entry will add the query to the search bar and filter the suggestions accordingly.
-    - To edit a saved search, in the "Saved searches" section, click {% octicon "pencil" aria-label="The pencil icon" %} to the right of the saved search.
-    - To delete a saved search, click {% octicon "trash" aria-label="The trash icon" %} to the right of the saved search.
+![Pop-up window, user creating saved search with name and query.](/assets/images/help/search/savedsearchquery.png)
+1. If you click again on the search bar, you can now see your saved search in the "Saved queries" section under the search bar. Clicking on a saved search entry will add the query to the search bar and filter the suggestions accordingly.
```

Comment:
```suggestion
1. To see your saved search, click the search bar. Your saved search will be in the "Saved queries" section. Clicking on a saved search entry will add the query to the search bar and filter the suggestions accordingly.
```

Just a couple more small changes to align this text with the [ordering recommended for procedural steps](https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#procedural-steps).